src/Storage/Adapter/RedisCluster.php

```diff
@@ -0,0 +1,315 @@
+<?php
```

Comment:
We will need some tests for this. We need to set up a redis cluster in the docker environment as a service to be able to run tests on it. I will help with that so reach out on discord.

Reply:
I'm on solo challenge, but appreciate. I have seen already codeception integration tests and how docker tests are run, hopefully can do this on my own and learn something new doing this.

Reply:
I added Cache\RedisCluster adapter and implemented almost all integration tests there. Still 4 tests that require maybe a little refactoring to kepp them DRY. Aiming for adjusting test in Storage\Adapter namespace too

Reply:
sonarcloud claim 12% duplication code in tests i only adjusted a bit. Don't know what should I do with this, sode marked as duplicate is not a code I wrote

Reply:
Don't worry about the duplication of code in the tests.